### PR TITLE
Added Preview Note and Release Note Link To How-Tos

### PR DESCRIPTION
**Pull request by luisquintanilla**
Added Preview Note and Release Note Link To How-Tos

Review Links:

File Type | File Name | Published Url
-- | -- | --
Content | docs/machine-learning/how-to-guides/consuming-model-ml-net.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/consuming-model-ml-net?branch=pr-en-us-11020
Content | docs/machine-learning/how-to-guides/determine-global-feature-importance-in-model.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/determine-global-feature-importance-in-model?branch=pr-en-us-11020
Content | docs/machine-learning/how-to-guides/inspect-intermediate-data-ml-net.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/inspect-intermediate-data-ml-net?branch=pr-en-us-11020
Content | docs/machine-learning/how-to-guides/load-data-from-mult-column-csv-ml-net.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/load-data-from-mult-column-csv-ml-net?branch=pr-en-us-11020
Content | docs/machine-learning/how-to-guides/load-data-from-multiple-files-ml-net.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/load-data-from-multiple-files-ml-net?branch=pr-en-us-11020
Content | docs/machine-learning/how-to-guides/load-data-from-text-file-ml-net.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/load-data-from-text-file-ml-net?branch=pr-en-us-11020
Content | docs/machine-learning/how-to-guides/load-non-file-training-data-ml-net.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/load-non-file-training-data-ml-net?branch=pr-en-us-11020
Content | docs/machine-learning/how-to-guides/matchup-app-infer-net.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/matchup-app-infer-net?branch=pr-en-us-11020
Content | docs/machine-learning/how-to-guides/normalizers-preprocess-data-ml-net.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/normalizers-preprocess-data-ml-net?branch=pr-en-us-11020
Content | docs/machine-learning/how-to-guides/single-predict-model-ml-net.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/single-predict-model-ml-net?branch=pr-en-us-11020
Content | docs/machine-learning/how-to-guides/train-cross-validation-ml-net.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/train-cross-validation-ml-net?branch=pr-en-us-11020
Content | docs/machine-learning/how-to-guides/train-model-categorical-ml-net.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/train-model-categorical-ml-net?branch=pr-en-us-11020
Content | docs/machine-learning/how-to-guides/train-model-textual-ml-net.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/train-model-textual-ml-net?branch=pr-en-us-11020
Content | docs/machine-learning/how-to-guides/train-regression-model-ml-net.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/train-regression-model-ml-net?branch=pr-en-us-11020
Content | docs/machine-learning/how-to-guides/use-gams-for-model-explainability.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/use-gams-for-model-explainability?branch=pr-en-us-11020
Content | docs/machine-learning/how-to-guides/verify-model-quality-ml-net.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/verify-model-quality-ml-net?branch=pr-en-us-11020

### CODE REVIEW

File: docs/machine-learning/how-to-guides/consuming-model-ml-net.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ ms.custom: mvc,how-to
 ---
 # Operationalize a trained machine learning model in apps - ML.NET
 
+> [!NOTE]
+> This topic refers to ML.NET, which is currently in Preview, and material may be subject to change. For more information, visit [the ML.NET introduction](https://www.microsoft.com/net/learn/apps/machine-learning-and-ai/ml-dotnet).
+
+This how-to and related sample are currently using **ML.NET version 0.10**. For more information, see the release notes at the [dotnet/machinelearning github repo](https://github.com/dotnet/machinelearning/tree/master/docs/release-notes)
```
Comment (pkulikov) [inline docs/machine-learning/how-to-guides/consuming-model-ml-net.md]: Added the dot at the end of the paragraph:
```suggestion
This how-to and related sample are currently using **ML.NET version 0.10**. For more information, see the release notes at the [dotnet/machinelearning github repo](https://github.com/dotnet/machinelearning/tree/master/docs/release-notes).
```
@luisquintanilla this applicable to all other additions in this PR as well.

Comment (luisquintanilla) [reply docs/machine-learning/how-to-guides/consuming-model-ml-net.md]: Good catch @pkulikov . Thanks.

Comment (pkulikov) [inline docs/machine-learning/how-to-guides/consuming-model-ml-net.md]: Missed at first: "GitHub" spelling
```suggestion
This how-to and related sample are currently using **ML.NET version 0.10**. For more information, see the release notes at the [dotnet/machinelearning GitHub repo](https://github.com/dotnet/machinelearning/tree/master/docs/release-notes)
```

File: docs/machine-learning/how-to-guides/matchup-app-infer-net.md [hunk 2]
```diff
@@ -7,6 +7,11 @@ ms.custom: mvc,how-to
 ---
```
Comment (JRAlexander) [inline docs/machine-learning/how-to-guides/matchup-app-infer-net.md]: Can you check the date on this?

File: docs/machine-learning/how-to-guides/verify-model-quality-ml-net.md [hunk 3]
```diff
@@ -7,6 +7,11 @@ ms.custom: mvc,how-to
 ---
```
Comment (JRAlexander) [inline docs/machine-learning/how-to-guides/verify-model-quality-ml-net.md]: Can you can check this date as well?

**Review by JRAlexander — CHANGES_REQUESTED** [review]
Just a couple of small items. Thanks!

File: docs/machine-learning/how-to-guides/verify-model-quality-ml-net.md (hunk 3 above)
Comment (luisquintanilla) [reply docs/machine-learning/how-to-guides/verify-model-quality-ml-net.md]: Done. Updated all the dates on all the How-To's.

https://github.com/dotnet/docs/pull/11020/commits/3afca9423998cf0449779598cd56af5e444d79ac

**Review by JRAlexander — APPROVED** [review]

**Pull request merged**